drivers/metadata/postgres/metadata.go

```diff
@@ -193,6 +194,58 @@ FROM pg_catalog.pg_class c
 	return metadata.NewIndexColumnSet(results), nil
 }
 
+func (r metaReader) Triggers(f metadata.Filter) (*metadata.TriggerSet, error) {
+	qstr := `SELECT 
+    t.tgname, 
+    pg_catalog.pg_get_triggerdef(t.oid, true)
+FROM 
+    pg_catalog.pg_trigger t 
+    JOIN pg_catalog.pg_class c ON c.oid = t.tgrelid
+	LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace`
+	conds := []string{`(
+	NOT t.tgisinternal OR (t.tgisinternal AND t.tgenabled = 'D') 
+			OR 
+				EXISTS (SELECT 1 FROM pg_catalog.pg_depend WHERE objid = t.oid 
+			AND 
+				refclassid = 'pg_catalog.pg_trigger'::pg_catalog.regclass)
+	)`}
+	vals := []interface{}{}
+	if f.Schema != "" {
+		vals = append(vals, f.Schema)
+		conds = append(conds, fmt.Sprintf("n.nspname LIKE $%d", len(vals)))
+	}
+	if f.Parent != "" {
+		vals = append(vals, f.Parent)
+		conds = append(conds, fmt.Sprintf("c.relname LIKE $%d", len(vals)))
+	}
+	if f.Name != "" {
+		vals = append(vals, f.Name)
+		conds = append(conds, fmt.Sprintf("t.tgname LIKE $%d", len(vals)))
+	}
+	rows, closeRows, err := r.query(qstr, conds, "t.tgname", vals...)
+	if err != nil {
+		return nil, err
+	}
+	defer closeRows()
+
+	results := []metadata.Trigger{}
+	for rows.Next() {
+		rec := metadata.Trigger{}
+		err = rows.Scan(
+			&rec.Name,
```

Comment:
Since there's filtering by scheme and parent, can you add their names to `SELECT` and fill them here?